cmd/venom/run/cmd.go

```diff
@@ -46,7 +46,7 @@ var (
 	outputDir     string
 	libDir        string
 	stopOnFailure bool
-	verbose       int = 1 // Set the default value for verboseFlag
+	verbose       int
```

Comment:
I think this was an error, because running with and without a single `-v` had the same behaviour, so to make it verbose you had to actually put at least `-vv`

In addition, since single `-v` was actually not implemented, imho this is perfect to implement this new report mode

Reply:
@lowlighter  with -v, the file venom.log is written.

Reply:
i remember this bug fix, the default value is 1 so we should let the initialization to 1